specification.md

```diff
@@ -1080,10 +1080,10 @@ V1](https://flux-framework.readthedocs.io/projects/flux-rfc/en/latest/spec_25.ht
 which is itself a simplified version of the [canonical Flux jobspec
 format](https://flux-framework.readthedocs.io/projects/flux-rfc/en/latest/spec_14.html).
 
-A jobspec V1 YAML document SHALL consist of a dictionary defining the
+A job specification V1 YAML document SHALL consist of a dictionary defining the
```

Comment:
Were these changes from `jobspec` -> `job specification` still intended after the latest commit?

Reply:
That's a good catch - I'll revert.